web/src/main/angular/src/app/core/components/load-avg-max-chart/load-avg-max-chart-container.component.ts

```diff
@@ -0,0 +1,355 @@
+            ]
+            : [];
+    }
+
+    private makeDataOption(columns: PrimitiveArray[]): Data {
+        const keyList = columns.slice(1).map(([key]: PrimitiveArray) => key as string);
+
+        return {
+            x: 'x',
+            columns,
+            empty: {
+                label: {
+                    text: this.dataEmptyText
+                }
+            },
+            type: 'area-step',
+            colors: keyList.reduce((acc: {[key: string]: string}, curr: string, i: number) => {
+                return { ...acc, [curr]: this.chartColors[i] };
+            }, {}),
+            groups: [keyList],
+            order: null
+        };
+    }
+
+    private makeElseOption(yMax: number): {[key: string]: any} {
+        return {
+            padding: {
+                top: 20,
+                bottom: 10,
+                right: 25
+            },
+            axis: {
+                x: {
+                    type: 'timeseries',
+                    tick: {
+                        count: 6,
+                        show: false,
+                        format: (time: Date) => {
+                            return moment(time).tz(this.timezone).format(this.dateFormatMonth) + '\n' + moment(time).tz(this.timezone).format(this.dateFormatDay);
+                        }
+                    },
+                    padding: {
+                        left: 0,
+                        right: 0
+                    }
+                },
+                y: {
+                    tick: {
+                        count: 3,
+                        format: (v: number): string => this.convertWithUnit(v)
+                    },
+                    padding: {
+                        top: 0,
+                        bottom: 0
+                    },
+                    min: 0,
+                    max: yMax,
+                    default: [0, this.defaultYMax]
+                }
+            },
+            grid: {
+                y: {
+                    show: true
+                }
+            },
+            tooltip: {
+                order: '',
+                format: {
+                    value: (v: number) => this.addComma(v.toString())
+                }
+            },
+            transition: {
+                duration: 0
+            }
+        };
+    }
+
+    private addComma(str: string): string {
+        return str.replace(/(\d)(?=(?:\d{3})+(?!\d))/g, '$1,');
+    }
+
+    private convertWithUnit(value: number): string {
+        const unitList = ['', 'K', 'M', 'G'];
+
+        return [...unitList].reduce((acc: string, curr: string, i: number, arr: string[]) => {
+            const v = Number(acc);
+
+            return v >= 1000
+                ? (v / 1000).toString()
+                : (arr.splice(i + 1), Number.isInteger(v) ? `${v}${curr}` : `${v.toFixed(2)}${curr}`);
+        }, value.toString());
+    }
```

Comment:
Here may need to update to show time from milliseconds to seconds and do not know how for now.